django/db/migrations/loader.py

```diff
@@ -186,15 +186,20 @@ def build_graph(self):
         # This is just for dependency re-pointing when applying replacements,
         # so we ignore run_before here.
         reverse_dependencies = {}
-        for key, migration in normal.items():
+        reverse_dependencies_normal = {}
+        for key, migration in self.disk_migrations.items():
             for parent in migration.dependencies:
                 reverse_dependencies.setdefault(parent, set()).add(key)
+                if key in normal:
```

Comment:
I guess once I add a comment explaining line 189, a comment here might be redundant..